cmd/commands/purge_osd.go

```diff
@@ -0,0 +1,41 @@
+package command
+
+import (
+	"fmt"
+	"strconv"
+
+	purgeosd "github.com/red-hat-storage/odf-cli/pkg/purge_osd"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	"github.com/spf13/cobra"
+)
+
+var PurgeCmd = &cobra.Command{
+	Use:   "purge",
+	Short: "Call subcommands like 'osd'",
```

Comment:
As long as only one subcommand we could have specific help text
```suggestion
	Short: "Purge an OSD",
```